README.md

```diff
@@ -1 +1,211 @@
-# ruby_morse
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
- This logo should not be found in the final README file. You can replace the logo with your own or get rid of it.